Eevee.Sleep.Bot/Utils/DiscordMessageMaker.cs

```diff
@@ -171,4 +171,57 @@ public static class DiscordMessageMaker {
             .WithCurrentTimestamp()
             .Build();
     }
+
+    public static MessageComponent MakeRoleSelectButton(
+        TrackedRoleModel[] roles,
+        ButtonId buttonId
+    ) {
+        var builder = new ComponentBuilder();
+
+        foreach (var role in roles) {
+            builder.WithButton(
+                label: role.Name,
+                customId: ButtonInteractionInfoSerializer.Serialize(
+                    new ButtonInteractionInfo() {
+                        ButtonId = buttonId,
+                        CustomId = role.Id
+                    }
+                )
+            );
+        }
+
+        return builder.Build();
+    }
+
+    public static Embed MakeDeleteRoleResult(IUser user, ulong roleId) {
+        return new EmbedBuilder()
+            .WithColor(Colors.Success)
+            .WithAuthor(user)
+            .WithTitle("Role deleted from database")
+            .AddField("User", user.Mention)
+            .AddField("Role", MentionUtils.MentionRole(roleId))
+            .WithCurrentTimestamp()
+            .WithFooter($"ID: {user.Id}")
+            .Build();
+    }
+
+    public static Embed MakeTrackRoleResult(
+        ulong roleId,
+        ulong[] trackedRoles,
+        int RoleOwnedUsercount,
+        string message
+    ) {
+        var trackedRolesMessage = trackedRoles.Length == 0 ?
+            "None" :
+            string.Join(" / ", trackedRoles.Select(x => MentionUtils.MentionRole(x)));
+
+        return new EmbedBuilder()
+            .WithColor(Colors.Success)
```

Comment:
Change `message` to something more description-like, maybe an enum indicating the "type" of the change. Then, map that enum to a color, so tracked has `Colors.Success`, untracked has `Colors.Warning` or `Colors.Danger`.

Reply:
I grasped the idea of changing the color, but I need an example sentence for the `message`.
I can only produce English sentences that look like translations, so I think it will be uncomfortable again.

Reply:
Is this resolved?

Reply:
No, I'm having a hard time knowing what kind of message to send
(I'm mainly talking about the two messages below

Reply:
What calls `DiscordMessageMaker.MakeTrackRoleResult()` again? Let me see what it can be

Reply:
Actually I think just change `RoleOwnedUsercount` to `roleOwnedUsercount` (note the letter casing) then it's fine